plugins/search/src/components/SearchContext/SearchContext.tsx

```diff
@@ -74,6 +77,9 @@ export const SearchContextProvider = ({
   const [filters, setFilters] = useState<JsonObject>(initialState.filters);
   const [term, setTerm] = useState<string>(initialState.term);
   const [types, setTypes] = useState<string[]>(initialState.types);
+  const [open, setOpen] = useState<boolean>(false);
+  const toggleModal = (): void => setOpen(prevState => !prevState);
```

Comment:
you may want to wrap this in a `useCallback` - it's a nice service to your users that the function they consume is stable, so they can safely use it in useEffect dependency lists etc

Reply:
good point, updating!

Reply:
Nice catch, thanks for commenting, refactored now!